pepper-apis/src/main/java/org/broadinstitute/ddp/util/RouteUtil.java

```diff
@@ -237,11 +237,12 @@ public static UserAndStudy findUserAndStudyOrHalt(Handle handle, String userGuid
             LOG.warn(err.getMessage());
             throw ResponseUtil.haltError(HttpStatus.SC_NOT_FOUND, err);
         }
-        User user = handle.attach(UserDao.class).findUserByGuid(userGuid).orElseThrow(() -> {
+        User user = handle.attach(UserDao.class).findUserByGuid(userGuid).orElse(null);
+        if (user == null) {
```

Comment:
I think the `orElse(null)` pattern is preferred so the error handling portion is regular code instead of having to create a closure object for `orElseThrow()`.